proto/eventbus/v1/pubsub_api.pb.go

```diff
@@ -3,11 +3,11 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.1
-// 	protoc        (unknown)
-// source: eventbus/v1/pubsub_api.proto
+// 	protoc-gen-go v1.28.1
+// 	protoc        v5.27.1
+// source: proto/eventbus/v1/pubsub_api.proto
```

Comment:
@anna-cross why were these re-generated? the proto folder has buf files and this is what used to be generated, by runnung `cd proto && buf generate` this proto is not yet used btw, maybe back out these changes?

Reply:
The proto was fine, but you used a different tool to regenerate the proto. All you have to do is update the proto in there and run `buf generate` in the folder not protoc?

Reply:
Ok, used buf gen